cmd/build/helmify/replacements.go

```diff
@@ -67,7 +67,10 @@ var replacements = map[string]string{
     {{- range $key, $value := .Values.mutatingWebhookExemptNamespacesLabels}}
```

Comment:
nit: let's change it to `values`

```suggestion
    {{- range $key, $values := .Values.mutatingWebhookExemptNamespacesLabels}}
```